uint32 seq
time stamp
string name
lthmi_nav/IntMap map
geometry_msgs/Pose init_pose
---
